client/src/components/common/boost/BoostCalibration.tsx

```diff
@@ -0,0 +1,86 @@
+import React , {useCallback, useState} from 'react';
+import {Button, Card, InputGroup, FormControl} from 'react-bootstrap';
+import {setCalibration as sendCalibrationValue, resetCalibration} from 'api/common/powerModel';
+// import {useSensorData} from 'api/v2/sensors/data';
+
+/**
+ * Boost distance claibration input form
+ * 
+ * @returns {React.Component} Component
+ */
+function BoostCalibrationInput() {
```

Comment:
I don't mind strongly either way, but you could combine this component with the parent one. The code might be simpler once you start passing props around.

Reply:
Yeah that's something I had thought of, I just wasn't sure if I should go for more readable and modular code by having them separate or an easier function instead. From what I've thought so far, the only thing the parent `BoostCalibration` needs is the calibration value for which I can simply pass a function in `BoostCalibration` to `BoostCalibrationInput` that allows the parent to access the value. If this is all I'll need I though having more modular code would be better to test and more readable. let me know what you think!

Reply:
Since you can justify it I'm happy with you keeping it as is! My personal thoughts were that since the child component was only used exactly once, it actually makes it less readable as you loose the context of how the child relates to the parent unless you read the parent component first. As I said though, this is fine too!

Reply:
hmmm that actually almost convinces me to go with a single class. I feel like that will make it quite big, but maybe I can try it out and see whether it still is readable to not.